### DISCUSSION TITLE
Looking for replacement for reflashed sidekicks

### DISCUSSION THREAD

**Discussion by Mibix** [awawa-dev/HyperHDR#636] [Q&A]
I just finished my first HyperHDR project, coming from a Dreamscreen 4k w/ 2 sidekicks.  I have a HTPC with a 2070 super running HyperHDR.  I also have a PS5 so I wanted to have everything HDMI 2.1 and here is what I went with:

**60A/5V/300W switching PSU
Plasma 2040 PicoPi
SK6812 60LED/m cool white RGBW LED strip(s)
 EVCOO EZ-SP12H28S  2x1 HDMI 2.1 splitter
Llano 4x1 HDMI 2.1 switch
MS2130 usb3.0 capture card**

I have flashed the two Sidekicks with WLED and they work alright but the colors are just not nearly as accurate and they lag behind a bit.  I have the colors set to as close as I could to the main LED strips behind the 77" OLED but I'm looking for a better solution.  Could I do some directional lamps with some sort of smart bulbs?   Color accuracy is a bit more important than speed and I would prefer them to be wireless besides power.  The Bulbs face up in each corner of the room on both sides of the TV.

Attached is a very compressed video of its current state:
https://github.com/awawa-dev/HyperHDR/assets/7806070/e060a7a2-00d5-4b3b-9ed5-ff223b77da72

**Comment by awawa-dev** [comment]
Hi
In the case of directional lamps with high quality of light offered (and solid & fast API which is equally important), I personally only know Philips lamps, but I use them with the "candy gamma" option enabled in HyperHDR, so I enhance their natural profile because that's how I like them the most, although probably at the expense of the real color reproduction. But even they have a slight delay compared to a cable solution.

**Reply by Mibix** [reply]
Would I need a Hue hub for that as well or could you refer me to which lamps/bulbs you are talking about?

**Reply by awawa-dev** [reply]
I've got only small Philips Hue Play. On YouTube you can find videos of people using larger Philips directional lamps together with HyperHDR. Unfortunately, a Hue bridge is required. Fortunately, there are quite often sales for bridge kits with bulbs or lamps, and people who already have it and just want to expand the set sell it cheaply: I bought my new one for 50% RRP.

**Reply by Mibix** [reply]
Is it possible to get these to work? https://www.amazon.com/dp/B096WZXKZP

**Reply by awawa-dev** [reply]
There is not even support for UDP protocol https://govee-public.s3.amazonaws.com/developer-docs/GoveeDeveloperAPIReference.pdf It is not enough to put together a few hardware elements to make a lamp, but you also need good programmers to cooperate with engineers to design efficient API and this is rather impossible for such products. I wouldn't bother with it.

**Reply by Mibix** [reply]
Alright, I have some of these arriving today, hoping they will be better than the Dreamscreen Sidekicks. https://www.amazon.com/dp/B07GXB3S7Z
